MCprep_addon/materials/prep.py

```diff
@@ -423,7 +423,11 @@ class MCPREP_OT_swap_texture_pack(
 
 	@classmethod
 	def poll(cls, context):
-		if world_tools.get_exporter(context) is not None:
+		if world_tools.get_exporter(context) != world_tools.WorldExporter.Unknown:
```

Comment:
I think this was accidentally reverted. This should check for `None`, since that means no exporter has been detected

Reply:
It's self consistent with my changes above as I intentionally had updated it to always return Unknown in the branches that were previously None, but we're discussing above now I suppose

Reply:
Resolved above